src/main/java/org/codehaus/mojo/exec/ExecMojo.java

```diff
@@ -340,6 +416,42 @@ else if ( !StringUtils.isEmpty( argsProp ) )
                         IOUtil.close( outputStream );
                     }
                 }
+                else if (useMavenLogger)
+                {
+                    getLog().debug("Will redirect program output to Maven logger");
+                    final String parentThreadName = Thread.currentThread().getName();
+                    final String logSuffix = "[" + parentThreadName + "] ";
```

Comment:
more of a prefix instead of a suffix… but it always says `[main]` for me and as such is redundant and unnecessary in the output (makes it unnecessarily wider)…

Reply:
In Jenkins, they are `[pool-1-thread-1 for channel]`, making the output definitely too wide to read.

Reply:
@mirabilos Good point about prefix instead of suffix, my bad.  
As for always getting `[main]`, were you running Maven multi-threaded on a reactor that is bound to have 2 projects running in parallel with this plugin?  This option adds no value for single threaded builds, but can become an aid in tracing program output in large reactors with lots of threads. For example 1000+ projects running with 50+ threads.